sim/core/spell_resistances.go

```diff
@@ -32,16 +32,24 @@ func (spell *Spell) ResistanceMultiplier(sim *Simulation, isPeriodic bool, attac
 		return attackTable.GetArmorDamageModifier(spell)
 	}
 
+	if spell.SpellSchool.Matches(SpellSchoolNone) {
```

Comment:
Seems like we could put this panic in `unit.RegisterSpell` instead.

Reply:
I wasn't sure about healing or other spells, that might just not need a spell school. If you get here, you're a damaging spell.

Reply:
Moved, and guarded the check by DamageMultiplier != 0 || ThreatMultiplier != 0, as for ProcMask. In testing, the only remaining offender was Gift of Arthas, let's hope it stays like this.

Reply:
Healing spells actually use both of those multipliers but this is fine to submit. The healing stuff is very WIP still anyway.